addons/common/Cfg3DEN.hpp

```diff
@@ -129,14 +129,56 @@ class Cfg3DEN {
                 };
             };
         };
+
+        class GVAR(EditCodeMulti10_Init): GVAR(EditCodeMulti10) {
+            class Controls: Controls {
+                class Background: Background {};
+                class Title: Title {};
+                class Value: Value {};
+                class GVAR(ValueInit): GVAR(Value) {
+                    // Copies contents of editable init box into the hidden
+                    // variant and into CBA_Init attribute.
+                    // Automatically adds call-block wrapper to enable
+                    // the usage of local variables and return values.
+                    onKillFocus = QUOTE(\
+                        private _code = ctrlText (_this select 0);\
+                        (uiNamespace getVariable [ARR_2('GVAR(InitAttributeValue)', controlNull)]) ctrlSetText _code;\
+                        if (_code != '') then {\
+                            _code = 'call{' + _code + '}';\
+                        };\
+                        (ctrlParentControlsGroup (_this select 0) controlsGroupCtrl 100) ctrlSetText _code;\
```

Comment:
Don't use the UI name space, but a parent control / display instead. Whatever is more appropriate.
I don't see there part where it strips the `call wrapper` ?

Reply:
The onLoad code is inherited.